cerr/errors.go

```diff
@@ -110,10 +111,16 @@ var (
 	ErrUserNotFound = newErr("E10044", codes.NotFound, http.StatusNotFound, "user not found")
 	// Return if a user already exists
 	ErrUserAlreadyExists = newErr("E10045", codes.AlreadyExists, http.StatusConflict, "user already exists")
+	// Returned when authorization has failed or is not possible
+	ErrAuthorizationFailed = newErr("E10045", codes.AlreadyExists, http.StatusUnauthorized, "authorization failed")
```

Comment:
Do we have a better substitute for `codes.AlreadyExists`? For me, it doesn't with well with an `Unauthorized` response.